docs/content/tardigrade.md

```diff
@@ -314,4 +314,4 @@ See [rclone about](https://rclone.org/commands/rclone_about/)
 
 If you get errors like `too many open files` this usually happens when the default `ulimit` for system max open files is exceeded. Native Storj protocol opens a large number of TCP connections (each of which is counted as an open file). For a single upload stream you can expect 110 TCP connections to be opened. For a single download stream you can expect 35. This batch of connections will be opened for every 64 MiB segment and you should also expect TCP connections to be reused. If you do many transfers you eventually open a connection to most storage nodes (thousands of nodes).
 
-To fix these, please raise your system limits. You can do this issuing a `ulimit -n 65536` just before you run rclone, inside a bash script or inside the `$USER/.bashrc` file. If you need to change limits system-wide, this usually takes place at `/etc/sysctl.conf` and/or `/etc/security/limits.conf` on Linux, but please refer to your operating system manual.
+To fix these, please raise your system limits. You can do this issuing a `ulimit -n 65536` just before you run rclone, inside a bash script or inside the `$HOME/.bashrc` file. If you need to change limits system-wide, this usually takes place at `/etc/sysctl.conf` and/or `/etc/security/limits.conf` on Linux, but please refer to your operating system manual.
```

Comment:
Actually, this is more complex than simply `.bashrc`; it depends on whether a terminal is interactive or not (eg `.profile`). The rules are different for the ash shell (the default in some Ubuntu derivatives) and the busybox variant of ash (eg Alpine Linux). Probably best to leave it at hints and walk away!

Reply:
Oh no.. I just wanted to fix the path `$USER/.bashrc` to `$HOME/.bashrc`, which I assumed was the intention. 😭 But thanks for the input! 😄
Not my original text, so not sure how much I want to wreck it at this point.. perhaps I can try to insert that this is an example for interactive bash or something.. :thinking:
Thanks again.

Reply:
No problem. Just mention ulimits and drop any shell stuff.